src/plugins/visualize/public/application/components/visualize_listing.tsx

```diff
@@ -69,12 +69,12 @@ export const VisualizeListing = () => {
     chrome.setBreadcrumbs([
       {
         text: i18n.translate('visualize.visualizeListingBreadcrumbsTitle', {
-          defaultMessage: 'Visualize',
+          defaultMessage: 'Visualize Library',
         }),
```

Comment:
You may want to remove all those keys from the translations file to avoid having stale translations

Reply:
I've actually heard this both ways. It's not trouble for me to remove the changed translations, but apparently doing so can actually slow down the translation process. I may be mistaken on this, so will look further into it.